content/pipeline/docker/syntax/routing.md

```diff
@@ -28,7 +28,7 @@ steps:
   - go build
   - go test
 
-node:
+nodes:
```

Comment:
No, it's `node`:
https://github.com/drone-runners/drone-runner-docker/blob/master/engine/resource/pipeline.go#L33